index.js

```diff
@@ -1,8 +1,5 @@
-const { connected } = require('process');
-
 const fs = require('fs');
 const path = require('path');
-const os = require('os');
```

Comment:
some linters will warn you, but it's always good practice to remove unused code, in this case unused imports.